packages/preact/src/hooks/useObservable.spec.js

```diff
@@ -23,8 +24,12 @@ const computedSum = {
 };
 
 const increment = () => ({
-  apply: (cache) => {
-    cache.update("(a|b)", (v) => v + 1);
```

Comment:
This was that special update syntax you mentioned - the nee version is a little more verbose, but also very explicit. Do think the compactness was nice but think trading it away is a win. How did you feel about this?

Reply:
I think it is an okay tradeoff.